lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -71,20 +71,20 @@ def syms_to_strs(hsh)
       newhsh
     end
 
-    def get_metadata(namespace_name, pod_name, container_name)
+    def get_metadata(namespace_name, pod_name)
       begin
         metadata = @client.get_pod(pod_name, namespace_name)
         return if !metadata
         labels = syms_to_strs(metadata['metadata']['labels'].to_h)
         annotations = match_annotations(syms_to_strs(metadata['metadata']['annotations'].to_h))
         if @de_dot
           self.de_dot!(labels)
+          self.de_dot!(annotations)
         end
         kubernetes_metadata = {
             'namespace_name' => namespace_name,
             'pod_id'         => metadata['metadata']['uid'],
             'pod_name'       => pod_name,
-            'container_name' => container_name,
```

Comment:
even if we aren't using the container name for metadata, it would still be useful to have the container name as a log field